tinyagentos/routes/agent_deploy.py

```
@@ -133,6 +138,132 @@ def resolve_deploy_routing(request: Request, body: "DeployAgentRequest") -> "JSO
+        out = await asyncio.to_thread(
+            subprocess.run,
+            ["tailscale", "ip", "-4"],
+            capture_output=True,
+            text=True,
+            timeout=5,
+        )
+        ip = (out.stdout or "").strip().splitlines()
+        if out.returncode == 0 and ip and ip[0].strip():
+            return ip[0].strip()
+    except Exception:  # noqa: BLE001
+        pass
```

Ruff (0.15.18)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)